dvc/config.py

```diff
@@ -177,6 +177,13 @@ class Config(object):  # pylint: disable=too-many-instance-attributes
     SECTION_GDRIVE_CLIENT_ID = "gdrive_client_id"
     SECTION_GDRIVE_CLIENT_SECRET = "gdrive_client_secret"
     SECTION_GDRIVE_USER_CREDENTIALS_FILE = "gdrive_user_credentials_file"
+    SECTION_GDRIVE_SERVICE_ACCOUNT_EMAIL = "gdrive_service_account_email"
+    SECTION_GDRIVE_SERVICE_ACCOUNT_USER_EMAIL = (
```

Comment:
see my question re impersonating ... do we actually need this?

Reply:
`SERVICE_ACCOUNT_USER_EMAIL` will become optional parameter